Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1150,6 +1150,10 @@ public static partial class PythonOps {
             }
         }
 
+        public static int ClampToInt32(this long value) {
```

Comment:
Any specific reason to have this one public?

Reply:
No. Was meant to be `internal`.